### DISCUSSION TITLE
Make ** modifier to add styles to all descendants

### DISCUSSION THREAD

**Discussion by FragsterAt** [tailwindlabs/tailwindcss#14328] [Ideas]
The `*:xxx` modifier add `\*\:ххх > *` classes, the `**:xxx` should add `\*\*\:xxx *` classes

**Comment by tryoxiss** [comment]
What would be  the  use  case  for this?

**Reply by FragsterAt** [reply]
Styling any indirect elements, for example - padding/border in table cells. 

Or debugging with outline/border.

**Reply by tryoxiss** [reply]
Yeah that makes sense
